content/millennium/dstu2/conformance/structure-definition.md

```diff
@@ -69,9 +69,9 @@ Authorization is not required.
 
 ## List of StructureDefinitions
 
-ID                                                                                                                             | Description
--------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------
-[`patient-friendly-display`](http://fhir.cerner.com/millennium/dstu2/medications/medication-order/#custom-extensions)          | Display string suitable for patient viewing.  
-[`scheduling-location`](http://fhir.cerner.com/millennium/dstu2/scheduling/schedule/#custom-extensions)                        | Reference to the location of an appointment being scheduled.
-[`medication-statement-category`](http://fhir.cerner.com/millennium/dstu2/medications/medication-statement/#custom-extensions) | Category of an order.
-[`encounter-guarantor`](http://fhir.cerner.com/millennium/dstu2/encounters/encounter/#custom-extensions)                       | The guarantors of an encounter. Contains a reference to a Patient, Person, or Organization and a period when the guarantor is effective.
+ID                                                                                                                                  |  Description
+------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------
+[`patient-friendly-display`](https://fhir-ehr.cerner.com/dstu2/StructureDefinition/patient-friendly-display?_format=json)           |  Display string suitable for patient viewing.
+[`scheduling-location`](https://fhir-ehr.cerner.com/dstu2/StructureDefinition/scheduling-location?_format=json)                     |  Reference to the location of an appointment being scheduled.
+[`medication-statement-category`](https://fhir-ehr.cerner.com/dstu2/StructureDefinition/medication-statement-category?_format=json) |  Category of an order.
+[`encounter-guarantor`](https://fhir-ehr.cerner.com/dstu2/StructureDefinition/encounter-guarantor?_format=json)                     |  The guarantors of an encounter. Contains a reference to a Patient, Person, or Organization and a period when the guarantor is effective.
```

Comment:
how about putting these links at the bottom of the file instead of in the table?

Reply:
Made the change here: [Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/2bcc448db093c8c5488198f21d9937fba354d60e#diff-eeb4d69334618c2654f5aee38ad9e21cL72)